opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -868,6 +947,51 @@ class EclMaterialLawManager
         }
     }
 
+    template <class Container>
+    void readGasWaterEffectiveParameters_(Container& dest,
+                                        const Opm::EclipseState& eclState,
+                                        unsigned satRegionIdx)
+    {
+        if (!hasGas || !hasWater)
+            // we don't read anything if either the gas or the water phase is not active
+            return;
+
+        dest[satRegionIdx] = std::make_shared<GasWaterEffectiveTwoPhaseParams>();
+
+        auto& effParams = *dest[satRegionIdx];
+
+        const auto& tableManager = eclState.getTableManager();
+
+        switch (eclState.runspec().saturationFunctionControls().family()) {
+        case SatFuncControls::KeywordFamily::Family_I:
+        {
+            // No Family I applicable in this case
```

Comment:
Doesn't `SLGOF` model this case?

Reply:
I believe not as SLGOF  does not allow for specification of gas-water capillary pressure.  For gas-water systems also eclipse requires specification of saturation functions for gas-water only by family II keywords

Reply:
>> Doesn't `SLGOF` model this case?
>
> I believe not as `SLGOF` does not allow for specification of gas-water capillary pressure.

Yes you're right, sorry about the confusion.